devel/floppy/Portfile

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+PortGroup       compilers 1.0
+
+name            floppy
+version         7.0
+revision        0
+categories      devel
+platforms       darwin
+license         Noncommercial
+maintainers     {@kamischi web.de:karl-michael.schindler} openmaintainer
+homepage        http://www.netlib.org/floppy/
+master_sites    ${homepage}
+distfiles       ${name}7.tgz
+dist_subdir     ${name}/${version}
+extract.mkdir   yes
+description     Fortran source code analyzer
+
+long_description {
```

Comment:
In both ports, the long description should not be in a set of `{` curly braces `}`.